donald_wade/week_02/styles.css

```diff
@@ -0,0 +1,164 @@
+* {
+  /*border: 1px dotted red;*/
+  box-sizing: border-box;
+}
+
+
+body {
+
+  padding: 2%;
+  background-color: black;
+  font-family: 'Vollkorn', serif;
+}
+
+.everything {
+  background-color: black;
+  color: white;
+  margin: none;
+  min-width: 100vw;
+  min-height: 100vh;
+
+}
+
+.topscreen {
+  width: 50vw;
+  /*height: 100vh;*/
+  margin: auto;
+  display: inline;
+  color: white;
+  background-color: black;
+}
+
+.secondpage {
+  width: 50vw;
+  /*height: 100vh;*/
+  margin: auto;
+  background-color: black;
+  color: white;
+  display: block;
+}
+
+.heading {
+  text-align: center;
+  font-family: 'Abril Fatface', cursive;
+  font-size: 3em;
+  background-color: black;
+  color: white;
+  /*width: 50vw;*/
+  margin-left: 20vw;
+
+}
+
+
+.appearing {
+  opacity: 0;
+  transition: all 0.8s ease;
+  cursor: text;
+}
+
+.appearing:hover {
+  opacity: 1;
+}
+
+nav {
+  /*padding: 10px;*/
+
+
+}
+.sidebar {
+  padding-top: 10%;
+  padding-left: 2%;
+  color: white;
+  min-width: 20vw;
```

Comment:
nice to see min-width here, want to note that quite useful to use max-width as well, as if you put lots of content your block will grow to any width it need if you use only min-width! Be aware of it